calabash-cucumber/lib/calabash-cucumber/usage_tracker.rb

```diff
@@ -56,6 +57,21 @@ def post_usage_async
       private
 
       # @!visibility private
+      def preferences
+        @preferences ||= Calabash::Cucumber::Preferences.new
+      end
+
+      # @!visibility private
+      def user_id
+        @user_id ||= preferences.user_id
+      end
+
+      # @!visibility private
+      def allowed_to_track
+        @allowed_to_track ||= preferences.usage_tracking
+      end
+
+      # @!visibility private
       def self.track_usage?
         @@track_usage && !self.xtc?
```

Comment:
@@track_usage is for us to use.

For example, I am setting this in the spec/spec_helper.rb and I will probably set it the support/hooks.rb of the repos we own.

Savvy users can also use this variable to control usage-tracking on a per project basis.

The vars and methods are doc'd private so we can add or remove them later.